test/unit/adapters/http.js

```diff
@@ -1068,15 +1050,14 @@ describe('supports http with nodejs', function () {
       }).catch(function (err) {
         error = err;
         failure = true;
-      }).finally(function () {
+      }).then(function () {
```

Comment:
A promise's `finally` isn't really the right place to throw... I changed it to be consistent with all other assertions as above.